src/test/java/tdl/datapoint/video/VideoMergingAcceptanceTest.java

```diff
@@ -152,6 +148,15 @@ public void upload_when_accumulator_video_exists() throws Exception {
 
     //~~~~~~~~~~ Helpers ~~~~~~~~~~~~~`
 
+    private String generateUniqueParticipant() {
```

Comment:
This method does more than it says. I was not expecting the comapreVideos and accumulator destination to be instantiated here.

Reply:
Reverted changes